rubicon/repository/base.py

```diff
@@ -522,9 +537,11 @@ def get_dataframe_data(self, project_name, dataframe_id, experiment_id=None):
         )
 
         try:
-            df = self._read_dataframe(dataframe_data_path)
+            df = self._read_dataframe(dataframe_data_path, kind)
         except FileNotFoundError:
-            raise RubiconException(f"No data for dataframe with id `{dataframe_id}` found.")
+            raise RubiconException(
+                f"No data for dataframe with id `{dataframe_id}` found. This might have happened if you forgot to set `kind='dask'` when trying to read dask dataframe."
```

Comment:
```suggestion
                f"No data for dataframe with id `{dataframe_id}` found. This might have "
                "happened if you forgot to set `kind='dask'` when trying to read a `dask` dataframe."
```